crowbar_framework/spec/fixtures/data_bags/bc-template-provisioner.json

```diff
@@ -0,0 +1,118 @@
+{
+  "default_user": "crowbar",
+  "default_password_hash": "$1$BDC3UwFr$\/VqOWN1Wi6oM0jiMOjaPb.",
+  "access_keys": "ssh-dss AAAAB3NzaC1kc3MAAACBAI0hfNVGSr9X6FTv1ogc8sCJkkhdSKiz6BBhm68bZiNzvXLgoNuCN89hC\/RQmW85wfyi29SILsrgMtjGUBdJ9Krf8t3cNIkTJFZem2eCZuNQzNMb2e1h3u7r1\/LHQ+Cqp9FUMfMzNnkvKSkfQaaoxsORP+D4390WohC69lm2T+fDAAAAFQCDEURhjjzLWXkgoytAN5QS3sZp1QAAAIBNh2Igo3gDLlu6VUKWXJLk5VCmr0tzmjLNVVk3xes\/501uUhx\/8tYaGbJbFwrE7O2PWnrWzrpMRC807jKHGg4B0hsC\/9mK6Hm8KDQCwinSyfpLD4lTJ6G03IxHLZGiQsGWYeEk\/bfByZjiiybSkmZKsSNVU7C9zd6KCv1Ise846AAAAIA02zKOgpTtQWSIYnrZY5XFnzDuHtTB+e+KHIb58DTl0E8Z\/u8oX6CaII10gNZkf4SJqC3G8jWFfQFBnR5PJSF5V5G5vebrxg27U4UO0PUpou3RhmlnQrq5K\/KB34VENM+jgifUoQyluqpyjihfWsqcsoBOsM278I37V57batkdhQ== injected-key-from-host",
+  "shell_prompt": "USER@HOST:CWD SUFFIX",
+  "enable_pxe": true,
+  "supported_oses": {
+    "ubuntu-12.04": {
+      "initrd": "install\/netboot\/ubuntu-installer\/amd64\/initrd.gz",
```

Comment:
The backslashes here are redundant and can be dropped.  Same throughout this whole file - forward slashes hardly ever need to be quoted.  Only time I can think of when they do, is when you are writing regexps in Ruby or Perl like `/foo\/bar/`, but even then you can avoid that via `%r{foo/bar}` or similar.